src/test_suite/test_runner_main.ts

```diff
@@ -162,155 +203,208 @@ export async function testRunnerMain(
+        let suite: TestSuite;
+        try {
+            suite = TestSuite.fromYamlString(fs.readFileSync(testFile, 'utf8'));
+        } catch (err) {
+            if (err.code === 'ENOENT' || err.code === 'EISDIR') {
+                const message = `Cannot open test file "${err.path}"`;
+                fail(message);
+                // Unreachable code exists for type assertion.
+                suite = undefined!;
+            } else {
+                throw err;
+            }
         }
-    }
 
-    if (runOneTest !== undefined) {
-        if (runOneTest >= 0 && runOneTest < suite.tests.length) {
-            suite = new TestSuite([suite.tests[runOneTest]]);
-        } else {
-            const message = `Invalid test number ${runOneTest}`;
-            fail(message);
+        if (runOneTest !== undefined) {
+            if (runOneTest >= 0 && runOneTest < suite.tests.length) {
+                suite = new TestSuite([suite.tests[runOneTest]]);
+            } else {
+                const message = `Invalid test number ${runOneTest}`;
+                fail(message);
+            }
         }
+
+        testSuites.push(suite);
     }
+
     if (brief) {
         console.log(' ');
         console.log('Displaying test utterances without running.');
-        for (const test of suite.filteredTests(suiteExpression)) {
-            console.log(`Test ${test.id}: ${test.comment}`);
-            for (const step of test.steps) {
-                const input = getYamlInputText(step, correctionLevel);
-                console.log(`  ${input}`);
+        testSuites.forEach(suite => {
+            for (const test of suite.filteredTests(suiteExpression)) {
+                console.log(`Test ${test.id}: ${test.comment}`);
+                for (const step of test.steps) {
+                    const input = getYamlInputText(step, correctionLevel);
+                    console.log(`  ${input}`);
+                }
+                console.log(' ');
             }
-            console.log(' ');
-        }
+        });
         console.log('Tests not run.');
         console.log('Exiting with failing return code.');
         process.exit(1);
     } else {
-        const aggregator = await suite.run(
-            processor,
-            world.catalog,
-            suiteExpression,
-            correctionLevel,
-            isomorphic,
-            !skipIntermediate
-        );
+        for (let i = 0; i < testSuites.length; i++) {
+            const suiteResults = await testSuites[i].run(
+                processor,
+                world.catalog,
+                suiteExpression,
+                correctionLevel,
+                isomorphic,
+                !skipIntermediate
+            );
 
-        if (markdown) {
-            const md = createMarkdown(aggregator);
-            console.log(md);
-        } else {
-            aggregator.print(showAll);
-        }
+            // If there are no results, explicitly call that out
```

Comment:
Copied comment, but doesn't actually call it out. Suggest either changing comment or calling it out?

Reply:
I'll remove the comment, I don't think we should be called it out